Source/Scene/Expression.js

```diff
@@ -226,6 +281,33 @@ define([
         return this._value;
     };
 
+    Node.prototype._evaluateVariableString = function(feature) {
+        var exp = this._value;
+        var result = "";
+        var i = exp.indexOf('${');
```

Comment:
We can optimize this by breaking up the string at runtime based on where the variables are, and then just concatenating here.  Basically, cache the `indexOf` calls at compile time.

We could even go farther to avoid memory allocations here (which will be implicit in the concatenating) but it let's wait until we profile since I don't expect these type of expressions to get evaluated often (most likely for UI, mouse over, etc.).